From 31d6d85beda8c2fd49d35c32ec8982fb7a09e4d9 Mon Sep 17 00:00:00 2001
From: Morten Tryfoss <morten@tryfoss.no>
Date: Tue, 25 Nov 2014 10:18:04 +0100
Subject: [PATCH] fixes for centos rpm specs

- updates for rtpengine and the new modules

(cherry picked from commit 16fd690d1857dfb16681d918f11ed0863cd3845c)
---
 pkg/kamailio/centos/6/kamailio.spec | 12 ++++++++++--
 1 file changed, 10 insertions(+), 2 deletions(-)

diff --git a/pkg/kamailio/centos/6/kamailio.spec b/pkg/kamailio/centos/6/kamailio.spec
index 3fde15f..6e5ad2e 100644
--- a/pkg/kamailio/centos/6/kamailio.spec
+++ b/pkg/kamailio/centos/6/kamailio.spec
@@ -566,7 +566,7 @@ fi
 %doc %{_docdir}/kamailio/modules/README.rr
 %doc %{_docdir}/kamailio/modules/README.rtimer
 %doc %{_docdir}/kamailio/modules/README.rtpproxy
-%doc %{_docdir}/kamailio/modules/README.rtpproxy-ng
+%doc %{_docdir}/kamailio/modules/README.rtpengine
 %doc %{_docdir}/kamailio/modules/README.sanity
 %doc %{_docdir}/kamailio/modules/README.sca
 %doc %{_docdir}/kamailio/modules/README.sdpops
@@ -603,6 +603,10 @@ fi
 %doc %{_docdir}/kamailio/modules/README.xhttp_rpc
 %doc %{_docdir}/kamailio/modules/README.xlog
 %doc %{_docdir}/kamailio/modules/README.xprint
+%doc %{_docdir}/kamailio/modules/README.jsonrpc-s
+%doc %{_docdir}/kamailio/modules/README.nosip
+%doc %{_docdir}/kamailio/modules/README.tsilo
+
 
 %dir %attr(-,kamailio,kamailio) %{_sysconfdir}/kamailio
 %config(noreplace) %{_sysconfdir}/kamailio/*
@@ -706,7 +710,7 @@ fi
 %{_libdir}/kamailio/modules/rr.so
 %{_libdir}/kamailio/modules/rtimer.so
 %{_libdir}/kamailio/modules/rtpproxy.so
-%{_libdir}/kamailio/modules/rtpproxy-ng.so
+%{_libdir}/kamailio/modules/rtpengine.so
 %{_libdir}/kamailio/modules/sanity.so
 %{_libdir}/kamailio/modules/sca.so
 %{_libdir}/kamailio/modules/sdpops.so
@@ -743,6 +747,10 @@ fi
 %{_libdir}/kamailio/modules/xhttp_rpc.so
 %{_libdir}/kamailio/modules/xlog.so
 %{_libdir}/kamailio/modules/xprint.so
+%{_libdir}/kamailio/modules/jsonrpc-s.so
+%{_libdir}/kamailio/modules/nosip.so
+%{_libdir}/kamailio/modules/tsilo.so
+
 
 %{_sbindir}/kamailio
 %{_sbindir}/kamctl
-- 
2.1.3

